test/runner/simulator.go

```diff
@@ -79,91 +82,187 @@ func (c *ClusterController) createSimulatorConfigMap(name string, config *Simula
-				"simulator": name,
+func (c *ClusterController) createSimulatorPod(simType string, name string) error {
+
+	pod := &corev1.Pod{}
+
+	if strings.Compare(simType, "onos") == 0 {
+
+		pod = &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: c.clusterID,
+				Labels: map[string]string{
+					"type":      "simulator",
+					"simulator": name,
+				},
 			},
-		},
-		Spec: corev1.PodSpec{
-			Containers: []corev1.Container{
-				{
-					Name:            "device-simulator",
-					Image:           "onosproject/device-simulator:latest",
-					ImagePullPolicy: corev1.PullIfNotPresent,
-					Ports: []corev1.ContainerPort{
-						{
-							Name:          "gnmi",
-							ContainerPort: 10161,
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:            "onos-device-simulator",
+						Image:           "onosproject/device-simulator:latest",
+						ImagePullPolicy: corev1.PullIfNotPresent,
+						Ports: []corev1.ContainerPort{
+							{
+								Name:          "gnmi",
+								ContainerPort: 10161,
+							},
 						},
-					},
-					ReadinessProbe: &corev1.Probe{
-						Handler: corev1.Handler{
-							TCPSocket: &corev1.TCPSocketAction{
-								Port: intstr.FromInt(10161),
+						ReadinessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								TCPSocket: &corev1.TCPSocketAction{
+									Port: intstr.FromInt(10161),
+								},
 							},
+							InitialDelaySeconds: 5,
+							PeriodSeconds:       10,
 						},
-						InitialDelaySeconds: 5,
-						PeriodSeconds:       10,
-					},
-					LivenessProbe: &corev1.Probe{
-						Handler: corev1.Handler{
-							TCPSocket: &corev1.TCPSocketAction{
-								Port: intstr.FromInt(10161),
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								TCPSocket: &corev1.TCPSocketAction{
+									Port: intstr.FromInt(10161),
+								},
+							},
+							InitialDelaySeconds: 15,
+							PeriodSeconds:       20,
+						},
+						VolumeMounts: []corev1.VolumeMount{
+							{
+								Name:      "config",
+								MountPath: "/etc/simulator/configs",
+								ReadOnly:  true,
 							},
 						},
-						InitialDelaySeconds: 15,
-						PeriodSeconds:       20,
 					},
-					VolumeMounts: []corev1.VolumeMount{
-						{
-							Name:      "config",
-							MountPath: "/etc/simulator/configs",
-							ReadOnly:  true,
+				},
+				Volumes: []corev1.Volume{
+					{
+						Name: "config",
+						VolumeSource: corev1.VolumeSource{
+							ConfigMap: &corev1.ConfigMapVolumeSource{
+								LocalObjectReference: corev1.LocalObjectReference{
+									Name: name,
+								},
+							},
 						},
 					},
 				},
 			},
-			Volumes: []corev1.Volume{
-				{
-					Name: "config",
-					VolumeSource: corev1.VolumeSource{
-						ConfigMap: &corev1.ConfigMapVolumeSource{
-							LocalObjectReference: corev1.LocalObjectReference{
-								Name: name,
+		}
+
+	} else if strings.Compare(simType, "stratum") == 0 {
```

Comment:
define `"onos"` and `"stratum"` as constants

Reply:
will do.